index.html

```diff
@@ -401,6 +403,12 @@ <h3>Retail</h3>
 that contains proof of legitimacy. After having verified that the shop is
 legit, he can purchase his gift.
         </dd>
+        <dt>
+<udef>R.4 Bona-fide shopper</udef>
+        </dt>
+        <dd>
+David owns a restaurant and has registered with a low cost wholesaler to purchase provisions in bulk. The wholesaler has issued David with a credential to prove that he is entitled to enter the warehouse in order to purchase goods that are not available to the general public. The credential is marked "non-transferable" to stop David passing the credential to his friends to allow them to purchase low cost provisions.
+        </dd>
```

Comment:
```suggestion
        <p class="shortUcContributor">Contributed by David Chadwick</p>
        </dd>
```

Feel free to change to your company, if you prefer.